signalling/room.go

```diff
@@ -0,0 +1,124 @@
+package signalling
+
+import (
+	"errors"
+	"log"
+	"sync"
+)
+
+var ErrPeerNotFound = errors.New("peer not found")
```

Comment:
Maybe something more descriptive of where the error occurred.
var ErrPeerNotFound = errors.New("[Standard] Peer Not Found")